.github/workflows/pr-manual-merge.js

```diff
@@ -0,0 +1,64 @@
+const { slackNotification, getLocalConfigs, ZERO_IMPACT_PREFIX } = require('./helpers.js');
+
+const LABELS = {
+  zeroImpact: 'zero-impact',
+};
+
+async function main({ github = getLocalConfigs().github, context = getLocalConfigs().context } = {}) {
+    const { payload = {} } = context;
+    const { pull_request } = payload;
+    if (!pull_request) {
+        console.log("No pull_request found in context payload. Skipping notification.");
+        return;
+    }
+
+    const { number, title, html_url, base = {} } = pull_request;
+    const { ref: baseRef } = base;
+    const isStage = baseRef === 'stage';
+    const prefix = isStage
+        ? ':merged: PR merged to stage:'
+        : ':rocket: Production release:';
+    console.log(`Sending notification for PR #${number}: ${title}`);
+    await slackNotification(
+      `${prefix} <${html_url}|#${number}: ${title}>.`,
+      process.env.MILO_RELEASE_SLACK_WH
+    ).catch(e => console.error("Error sending Slack notification:", e.message))
+    
+    if (baseRef === 'stage') {
+        await updateStageToMainPR(github, context, pull_request);
+    }
```

Comment:
Nit: I would pull this into the `updateStageToMainPR` method as `if(baseRef !== 'stage') return`

It's firstly a good strategy on avoiding nesting if/else cases for more complex methods, secondly, if any other method would call `updateStageToMainPR` - you would need to duplicate this check across all instances where a method is called.
So it's always better to put checks closer to where they belong